__tests__/__snapshots__/index.spec.js.snap

```diff
@@ -0,0 +1,194 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`babel-preset-amex allows options to be passed to plugins 1`] = `
+Object {
+  "plugins": Array [
+    [Function],
+    [Function],
+    [Function],
+    [Function],
+    [Function],
+  ],
+  "presets": Array [
+    Array [
+      Object {
+        "default": [Function],
+        "getModulesPluginNames": [Function],
+        "getPolyfillPlugins": [Function],
+        "isPluginRequired": [Function],
+        "transformIncludesAndExcludes": [Function],
+      },
+      Object {
+        "exclude": Array [
+          "@babel/plugin-transform-regenerator",
+        ],
+        "targets": Object {
+          "browsers": Array [
+            "last 2 versions",
+            "not dead",
+          ],
+          "node": "current",
+        },
+      },
+    ],
+    Array [
+      Object {
+        "default": [Function],
+      },
+      Object {},
+    ],
+  ],
+}
+`;
+
+exports[`babel-preset-amex returns modern preset for env and option 1`] = `
+Object {
+  "plugins": Array [
+    [Function],
+    [Function],
+    [Function],
+    [Function],
+    [Function],
+  ],
+  "presets": Array [
+    Array [
+      Object {
+        "default": [Function],
+        "getModulesPluginNames": [Function],
+        "getPolyfillPlugins": [Function],
+        "isPluginRequired": [Function],
+        "transformIncludesAndExcludes": [Function],
+      },
+      Object {
+        "targets": Object {
+          "browsers": Array [
+            "last 2 Chrome versions",
+            "last 2 ChromeAndroid versions",
+            "last 2 Edge versions",
+            "last 2 Firefox versions",
+            "last 2 iOS versions",
+            "last 2 Safari versions",
+          ],
+          "node": "current",
+        },
+      },
+    ],
+    Array [
+      Object {
+        "default": [Function],
+      },
+      Object {},
+    ],
+  ],
+}
+`;
+
+exports[`babel-preset-amex returns modern preset for env and option 2`] = `
+Object {
+  "plugins": Array [
+    [Function],
+    [Function],
+    [Function],
+    [Function],
+    [Function],
```

Comment:
I know this is a pain but can we get some sort of string label on what these plugins are in the snapshot for transparency purposes?

Reply:
not sure thats an option with snapshots, unable to find anything in the docs for that

Reply:
We can mock the plugins and turn them into strings. Since we're not testing the plugins themselves but the configuration, this is definitely an option. I have done this on occasion elsewhere.

Reply:
updated 8ddaaa3